checkov/cloudformation/graph_builder/local_graph.py

```diff
@@ -62,5 +87,180 @@ def _create_section_vertices(self, file_path: str, file_conf: dict, section: Clo
+        """ might be one of the 2 following notations:
+         1st: { "Fn::GetAtt" : [ "logicalNameOfResource", "attributeName" ] }
+         2nd: { "!GetAtt" : "logicalNameOfResource.attributeName" } """
+        target_id = None
+
+        # Fn::GetAtt notation
+        if isinstance(value, list) and len(value) == 2 and (self._is_resource(template, value[0])):
+            target_id = value[0]
+
+        # !GetAtt notation
+        if isinstance(value, (six.string_types, six.text_type)) and '.' in value:
+            if self._is_resource(template, value.split('.')[0]):
+                target_id = value.split('.')[0]
+
+        return target_id
+
+    def _fetch_ref_target_id(self, template, value) -> int:
+        target_id = None
+        # value might be a string or a list of strings
+        if isinstance(value, (six.text_type, six.string_types, int)) \
+                and ((self._is_resource(template, value)) or (self._is_parameter(template, value))):
+            target_id = value
+        return target_id
+
+    def _fetch_findinmap_target_id(self, template, value) -> int:
+        target_id = None
+        # value = [ MapName, TopLevelKey, SecondLevelKey ]
+        if isinstance(value, list) and len(value) == 3 and (self._is_mapping(template, value[0])):
+            target_id = value[0]
+        return target_id
+
+    def _add_fn_sub_connections(self):
+        for file_path, template in self._templates.items():
+            # add edges for "Fn::Sub" tags. E.g. { "Fn::Sub": "arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:vpc/${vpc}" }
+            sub_objs = template.search_deep_keys(IntrinsicFunctions.SUB.value)
+            for sub_obj in sub_objs:
+                sub_parameters = []
+                sub_parameter_values = {}
+                source_id, value, attributes = self._extract_source_value_attrs(sub_obj)
+
+                if isinstance(value, list):
+                    if not value:
+                        continue
+                    if len(value) == 2:
+                        sub_parameter_values = value[1]
+                    sub_parameters = self._find_fn_sub_parameter(value[0])
+                elif isinstance(value, (six.text_type, six.string_types)):
+                    sub_parameters = self._find_fn_sub_parameter(value)
+
+                for sub_parameter in sub_parameters:
+                    if sub_parameter not in sub_parameter_values:
+                        if '.' in sub_parameter:
+                            sub_parameter = sub_parameter.split('.')[0]
+                        origin_vertex_index, dest_vertex_index, label = self._extract_origin_dest_label(
+                            file_path, source_id, sub_parameter, attributes)
+                        if origin_vertex_index is None or dest_vertex_index is None:
+                            continue
+                        self._create_edge(origin_vertex_index, dest_vertex_index, label)
```

Comment:
```suggestion
                        if origin_vertex_index is not None and dest_vertex_index is not None:
                            self._create_edge(origin_vertex_index, dest_vertex_index, label)
```
same as @nimrodkor suggested before, looks cleaner 🙂